dashboard/lib/pd/payment/payment_factory.rb

```diff
@@ -15,12 +15,9 @@ def self.get_calculator_class(workshop)
           Pd::Workshop::COURSE_CS_IN_A,
           Pd::Workshop::COURSE_CS_IN_S
         ].include?(workshop.course)
-
-        return PaymentCalculatorCounselorAdmin if [
-          Pd::Workshop::COURSE_COUNSELOR,
-          Pd::Workshop::COURSE_ADMIN
-        ].include?(workshop.course)
       else
+        return PaymentCalculatorCounselorAdmin if Pd::Workshop::COURSE_ADMIN_COUNSELOR == workshop.course
+
```

Comment:
The COURSE_ADMIN_COUNSELOR logic was moved in the `else` statement since they are never labeled as `funded`. I felt it was okay to remove the `PaymentCalculatorCounselorAdmin` in reference to `COURSE_COUNSELOR` and `COURSE_ADMIN` since there won't be a need for new payment calculators for deprecated workshops I believe.